### ISSUE TITLE
Terms and conditions for user-generated content

### ISSUE THREAD

**Issue by StatusSceptre** [status-im/status#50]
## Problem
The app currently doesn't include or direct to terms and conditions with a definition of acceptable user-generated content. Play store requires the app to have users accept terms and conditions on UGC. [App store](https://developer.apple.com/app-store/review/guidelines/#user-generated-content) does not require a T&C to be accepted.

[Play store](https://support.google.com/googleplay/android-developer/answer/9876937):
> - require that users accept the app's terms of use and/or user policy before users can create or upload UGC;
> - define objectionable content and behaviors (in a way that complies with Play’s Developer Program Policies), and prohibit them in the app’s terms of use or user policies

Introducing profile pictures in release 1.10 (across all chats, incl public chats), makes it more likely that it will become apparent during a Play store review that no terms and conditions are in place

#### Actions
- [ ] Update first onboarding screen copy and add link
     - "Status does not collect or profit from your personal data. By continuing, you agree with the privacy policy **and terms and conditions**"
- [ ] Add webpage to host terms and conditions e.g. https://status.im/termsandconditions (to be defined)
- [ ] Write application terms and conditions
     - applying only to users downloading and installing the mobile app through Google Play store

cc @guylouis

**Comment by guylouis** [comment]
- [ ] also we should inform existing users upgrading their app that they do accept T&Cs. With a pop-up for instance.

**Comment by hesterbruikman** [comment]
Entry points in the applications:

![Intro1](https://user-images.githubusercontent.com/35233054/103629135-3d4eae80-4f40-11eb-8430-949a8609d4d6.png)
![About](https://user-images.githubusercontent.com/35233054/103629138-3de74500-4f40-11eb-9be3-66341e80d8c1.png)

**Comment by hesterbruikman** [comment]
fyi @iurimatias @simonricoo this issue is also relevant for Desktop. Once Terms and Conditions are available, the same changes should be made on Desktop. Adjust copy on onboarding screen to include link to Terms and Conditions and add a list item and link to Settings > About

**Comment by simonricoo** [comment]
Im going to add that too! Thanks

**Comment by simonricoo** [comment]
@guylouis good point, if we updated those docs we should definitely request an approval from existing users.

**Comment by errorists** [comment]
<img width="377" alt="Screenshot 2021-01-08 at 09 47 44" src="https://user-images.githubusercontent.com/4932774/103993899-91eb6700-5196-11eb-8d72-2bf75cbeca1f.png">

Attaching my proposed design for this, obviously inspired by Signal's own.
